File: examples/input/sample.js
// Sample JavaScript file with logs
function calculateTotal(items) {
  console.log('Calculating total for', items.length, 'items');

  let total = 0;
  for (const item of items) {
    console.debug('Processing item:', item);
    total += item.price;
  }

  // @keep - This log should be preserved
  console.log('Final total calculation complete');

  console.error('This should be removed');
  debugger;

  return total;
}

// Commented log
// console.log('This is commented');

/* Block commented log
   console.warn('Block comment');
*/

console.info('End of file');

File: examples/input/sample.py
# Sample Python file with logs
def calculate_total(items):
    print('Calculating total for', len(items), 'items')
    
    total = 0
    for item in items:
        print(f'Processing item: {item}')
        total += item.price
    
    # @keep - This should be preserved
    print('Final total calculation complete')
    
    print('This should be removed')
    
    return total

# Commented log
# print('This is commented')

if __name__ == '__main__':
    print('Running main')

